tsconfig.json

```diff
@@ -1,33 +1,33 @@
 {
-  "compilerOptions": {
-    "baseUrl": ".",
-    "emitDecoratorMetadata": true,
-    "experimentalDecorators": true,
-    "lib": [
-      "es6",
-      "dom"
-    ],
-    "module": "commonjs",
-    "moduleResolution": "node",
-    "noImplicitAny": false,
-    "paths": {
-      "*": [
-        "./node_modules/tns-core-modules/*",
-        "./node_modules/*"
-      ]
+    "compilerOptions": {
```

Comment:
I think it reformatted it from using 2 spaces to 4 spaces. I believe typescript did this automatically as I did not touch it